testutil/integ.go

```diff
@@ -15,6 +15,7 @@
 package testutil
 
 import (
+	"log"
```

Comment:
Why not our internal logger?

Reply:
You mean https://github.com/abcxyz/pkg/tree/main/logging?

No need. For test output, we don't have to use structured logging. We expect an exit right away.

Reply:
Yea - our logger is for structured application level logging. That being said, it might be easier to just `return false` instead of logging a message.

Reply:
I vaguely remember a quick discussion about this - if we return `false` when the env var value is not a boolean, a typo (e.g. `truu`) could "sneakily" skip the test.